pkg/entities/set.go

```diff
@@ -43,28 +41,36 @@ const (
 // Do not expose set to IPFIX library user
 // Not creating any interface. Plan to use same struct for Template and Data sets
 
-type Set struct {
+type Set interface {
```

Comment:
Does this interface need mock for unit testing of collector and exporter?

Reply:
Yes, will add mock here.